01-js-intro-and-jquery/keypoints/1_7,6,8,9,10/resources/css/style.css

```diff
@@ -0,0 +1,16 @@
+
+.alias {
```

Comment:
If the class do not define any css attribute but it's used into the html just for some JS action it's not needed to add it to the css file. Just a thing to have in mind for future cases